src/app/core/services/auth.service.ts

```diff
@@ -4,22 +4,34 @@ import { Observable, of, throwError } from 'rxjs';
 import { IUser } from '../models/user';
 import { Router } from '@angular/router';
 import { regUsers } from '../../../assets/data/users';
+import { admin } from '../../../assets/data/admin';
 
 @Injectable()
 export class AuthService {
+  public administrator: IUser | undefined;
```

Comment:
It unnecessary field, admin is the same user as other ones, the only difference is its role. You don;t need any additional fields